internal/service/project/resource_project.go

```diff
@@ -220,8 +221,9 @@ func (r *projectRS) Schema(ctx context.Context, req resource.SchemaRequest, resp
 				},
 			},
 			"is_slow_operation_thresholding_enabled": schema.BoolAttribute{
-				Computed: true,
-				Optional: true,
+				Computed:           true,
+				Optional:           true,
+				DeprecationMessage: fmt.Sprintf(constant.DeprecationParamByVersion, "1.24.0"),
```

Comment:
not strictly related to PR, do we have a ticket for creating the new resource? we should make sure to prioritise so that customers have the alternative option available

Reply:
created: CLOUDP-280721